app/features/tasks.py

```diff
@@ -1 +1,56 @@
+import pickle 
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
 
+class tasks:
+
+    def __init__(self):
+        self.SCOPES = ['https://www.googleapis.com/auth/tasks']
+        self.creds = None
+        if os.path.exists('token_calendar_v3.pickle'):
+            with open('token_calendar_v3.pickle', 'rb') as token:
+                self.creds = pickle.load(token)
+            
+        if not self.creds or not self.creds.valid:
+            if self.creds and self.creds.expired and self.creds.refresh_token:
+                self.creds.refresh(Request())
+            else:
+                flow = InstalledAppFlow.from_client_secrets_file('credentials.json', 'https://www.googleapis.com/auth/tasks')
+                self.creds = flow.run_local_server(port=0)
+            
+            with open('token.pickle', 'wb') as token:
+                pickle.dump(self.creds, token)
+            
+        self.service = build('tasks', 'v1', credentials=self.creds)
+
+    def list_tasks(self):
+        results = self.service.tasklists().list().execute()
+        items = results.get('items', [])
+
+        if not items:
+            print('No task lists found.')
+        else:
+            print('Task lists:')
+            for item in items:
+                self.tasklist.append(u'{0} ({1})'.format(item['title'], item['id']))
```

Comment:
Go ahead and remove this, we don't need to itterate through the unused tasklist variable anymore